jans-cli-tui/cli_tui/plugins/010_auth_server/ssa.py

```
@@ -174,8 +174,7 @@ def get_ssa(self, search_str=''):
         async def coroutine():
             cli_args = {'operation_id': 'get-ssa', 'cli_object': self.cli_object}
             self.app.start_progressing(_("Retreiving ssa..."))
-            response = await get_event_loop().run_in_executor(self.app.executor, self.app.cli_requests, cli_args)
-            self.app.stop_progressing()
+            response = await common_data.app.run_config_api_operation(cli_args, msg)
```

Ruff (0.15.9)

[error] Undefined name `common_data`

(F821)